### PR TITLE
O3-2042: Configure E2E tests with playwright

### PR DESCRIPTION
**Pull request by anjula-sack**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

- In this pull request E2E end tests were configured using playwright. All the necessary documents are created. The test configurations are similar to the patient management configuration. 

## Screenshots

*None.*

## Related Issue

https://issues.openmrs.org/browse/O3-2042

## Other

*None.*

### CODE REVIEW

File: e2e/storageState.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      "domain": "localhost",
+      "path": "/openmrs",
+      "expires": -1,
+      "httpOnly": true,
+      "secure": true,
+      "sameSite": "Strict"
+    }
+  ],
+  "origins": []
+}
```
Comment (jayasanka-sack) [inline e2e/storageState.json:15]: This file should be ignored.

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: e2e/storageState.json (hunk 1 above)
Comment (anjula-sack) [reply e2e/storageState.json]: Done

**Review by jayasanka-sack — APPROVED** [review]
Thanks @anjula-sack !

**Comment by vasharma05** [comment]
Great work!
Thanks @anjula-sack !

**Pull request merged**